docs/root/version_history/current.rst

```diff
@@ -29,6 +29,7 @@ Removed Config or Runtime
 
 New Features
 ------------
+* config: added new runtime feature `envoy.features.enable_all_deprecated_features` that allows the use of all deprecated features.
```

Comment:
Can you please also discuss this new feature here? https://www.envoyproxy.io/docs/envoy/latest/faq/configuration/deprecation

cc @alyssawilk if there are any other locations we should document.

Reply:
I ended up updating this: https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/runtime#using-runtime-overrides-for-deprecated-features instead of the FAQ.
Please let me know if there's anything else that I've missed.